v1.0/coordinatePoint.schema.json

```diff
@@ -1,22 +1,22 @@
 {
   "_type": "https://openminds.ebrains.eu/sands/CoordinatePoint",
-  "description": "Structured information describing a coordinate point in a space",
+  "description": "Structured information describing a coordinate point in a defined space.",
   "required": [
     "coordinates",
     "coordinateSpace"
   ],
   "properties": {
     "coordinates": {
       "type": "array",
-      "description": "Coordinates that describe the coordinate point (x, y , z).",
+      "_instruction": "Enter the coordinates (x, y , z; 2D, with z = 0) that describe this coordinate point, defined in the unit of the coordinate space.",
```

Comment:
"Add the coordinates ([x, y, z]; z = 0 in 2D) that define the exact location of this point in the stated space."